packages/xprog-prensa/news/Topbar/component.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react'
+
+import { useComponentFromProps } from '../../hooks'
+import { Block } from '../../primitives/Block'
+import * as S from './styled'
+import { TopbarProps } from './types'
+
+export const Topbar: React.FC<TopbarProps> = (props) => {
+
+  const {
+    children,
+    css,
+    slotBottomCenter,
+    slotBottomLeft,
+    slotBottomRight,
+    slotMiddleCenter,
+    slotMiddleLeft,
+    slotMiddleRight,
+    slotTopCenter,
+    slotTopLeft,
+    slotTopRight
+  } = props
+
+  const SlotTopLeft: any = slotTopLeft
+  const SlotTopCenter: any = slotTopCenter
+  const SlotTopRight: any = slotTopRight
+  const SlotMiddleLeft: any = slotMiddleLeft
+  const SlotMiddleCenter: any = slotMiddleCenter
+  const SlotMiddleRight: any = slotMiddleRight
+  const SlotBottomLeft: any = slotBottomLeft
+  const SlotBottomCenter: any = slotBottomCenter
+  const SlotBottomRight: any = slotBottomRight
+
+  const renderTopContainer = () => {
+    if (slotTopCenter || slotTopLeft || slotTopRight) {
+      return (
+        <S.SlotTopContainer
+          className='pds-Topbar-SlotTopContainer'
+          css={css?.cssContainerTop}
+        >
+          {SlotTopLeft &&
```

Comment:
Será que os slots não precisam serem renderizados, mesmo que tenham conteúdo vazio? Penso no caso do alinhamento, para quando no Left for vazio e no Center e Right tiverem itens, o between deveria funcionar como se tivesse item no Left.